Cython/Compiler/ModuleNode.py

```diff
@@ -187,91 +187,94 @@ def h_entries(entries, api=0, pxd=0):
         h_extension_types = h_entries(env.c_class_entries)
         if h_types or h_vars or h_funcs or h_extension_types:
             result.h_file = replace_suffix_encoded(result.c_file, ".h")
-            h_code = Code.CCodeWriter()
```

Comment:
I'd keep this around, maybe name it `h_code_writer`. Referring to `globalstate.rootwriter` below seems to touch into internals of `GlobalState`. (Sorry if this seems inconsistent with accessing `.parts`, but we are creating the code writer ourselves here and thus own it, so why refer back to it through an object internal copy of the reference later?)